ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -13,6 +13,93 @@ oso_snapshot_aws_secret_access_key: "{{ osohm_snapshot_aws_secret_access_key }}"
 gcp_volume_user_creds: '{{ osohm_snapshot_gcp_creds | to_json }}'
 {% endif %}
 
+metric_sender_config:
+  host:
+    name: "{{ osohm_host_name }}"
+  zagg:
+    active: True
+    url: "{{ osohm_zagg_web_url }}"
+    user: "{{ osohm_default_zagg_server_user }}"
+    pass: "{{ osohm_default_zagg_server_password }}"
+    ssl_verify: "{{ osohm_zagg_verify_ssl }}"
+    verbose: False
+    debug: False
+  hawk:
+    active: False
+    url: "{{ osohm_hawk_web_url }}"
+    user: "{{ osohm_default_hawk_server_user }}"
+    pass: "{{ osohm_default_hawk_server_password }}"
+    ssl_verify: "{{ osohm_hawk_verify_ssl }}"
```

Comment:
@zgalor vars like osohm_zagg_* are passed into this oso_host_monitoring role, and the vars are defined in our private data hierarchy